ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.21__multi-nightly.yaml

```diff
@@ -2172,6 +2172,34 @@ tests:
     - chain: cert-manager-custom-api-ingress-cert
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
+- as: gcp-ipi-oidc-mac-no-actas-mini-perm-amd-f28-destructive
```

Comment:
@jianli-wei, the PR looks good to me generally, just one comment, I noticed _machine-api-controller (mac)_ was used as the name of variables or files, this is fine, but I think _machine-api (mapi)_ is more suitable for this case as everyone knows it's relevant to machine api operator, which is the namespace stores the secret.